docs/markdown/concepts-and-architecture.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Concepts and Architecture
+nav_order: 2
+has_children: true
+permalink: /concepts/
+---
+
+# Concepts and Architecture
+Snowbridge is a complex project with lots of moving pieces that can interact in various configurations. In order to develop a comprehensive understanding of how things work, this section tries to break down and run through those pieces into different concepts and components that should be easier to understand.
+
+## Layered Architecture
+Snowbridge has a layered architecture with a clear seperation between low level bridge functionality, trust functionality and application functionality.
+
+If you're familiar with the conventional [OSI communication model](https://en.wikipedia.org/wiki/OSI_model), our system is a similar simplified version.
```

Comment:
The OSI model actually turned out to be a dead-end and TCP/IP won out in the end. So maybe lets link to that instead?